src/components/Tooltip/index.tsx

```diff
@@ -0,0 +1,78 @@
+import type { FC, ReactNode } from 'react';
+
+import React from 'react';
+import * as TooltipPrimitive from '@radix-ui/react-tooltip';
+import classnames from 'classnames';
+import { CSS_PREFIX } from '../constants';
+import './Tooltip.scss';
+
+export const BASE_CLASSNAME = `${CSS_PREFIX}-tooltip`;
+
+export type TooltipProps = {
+  open?: TooltipPrimitive.TooltipProps['open'];
+  defaultOpen?: TooltipPrimitive.TooltipProps['defaultOpen'];
+  onOpenChange?: TooltipPrimitive.TooltipProps['onOpenChange'];
+  side: TooltipPrimitive.TooltipContentProps['side'];
+  align: TooltipPrimitive.TooltipContentProps['align'];
+  content?: ReactNode;
+  rootProps?: Partial<Omit<TooltipPrimitive.TooltipProps, 'open' | 'defaultOpen' | 'onOpenChange'>>;
+  triggerProps?: Partial<Omit<TooltipPrimitive.TooltipTriggerProps, 'asChild'>>;
+  contentProps?: Partial<Omit<TooltipPrimitive.TooltipContentProps, 'side' | 'align'>>;
+  arrowProps?: Partial<TooltipPrimitive.TooltipArrowProps>;
+  classNames?: {
+    triggerClass?: string;
+    contentClass?: string;
+    arrowClass?: string;
+  };
```

Comment:
In my opinion, this granularity is breaking the idea of how our design system should work. For example, devs should not be able to restyle the arrow, because the arrow should have the same design throughout the design system. Similarly, they shouldn't be able to change how the trigger works (through `triggerProps`) because the tooltip should work the same everywhere.

I think we should:
- Remove `classNames` and add `className: string` (although I'm not even sure if we'd need a `className` here at all)
- Remove the `xProps` props (`rootProps`, `triggerProps`, `contentProps`, `arrowProps`)

Happy for discussion on this one!